Change the linting sections in yaml to support molecule v3

whowutwut opened the pull request:
Resolves #109  

Going by the documentation here: https://molecule.readthedocs.io/en/latest/configuration.html#lint  I made the changes.  I'm unable to get my forked copy to show up on my TravisCI, so I will submit this PR and see whether it fixes the molecule step

whowutwut commented:
HAHA, well that snowballed, we have around 700 lines of failed linting 

<img width="826" alt="image" src="https://user-images.githubusercontent.com/10049070/75259173-9b095a80-57b5-11ea-9ad8-501f3d40803e.png">

Let me at least fix the low hanging fruit and see what happens...

whowutwut commented:
I think the change above should pass now, let's see the Travis Results....

@mew2057  Going to need you to take a closer look at the `test_local/playbook.yaml` file changes, as I'm not sure if I did the block logic correctly and/or the include_tasks/playbook at the end...   Thanks

whowutwut commented:
If I can get this Travis CI to run on the latest commit, I think it's failing on the next linter.

mew2057 commented:
Found the offending code in `travis.yml`:
```
- python hacks/change_deploy_image.py -i ${OPERATOR_LOCAL_IMAGE} --ifnotpresent
```
In particular it's this line in the script
```
yaml.dump(opobj, outfile, default_flow_style=False)
```

Working on  a solution, we need this script  for the tests to actually run, might need  to rewrite it.

mew2057 commented:
I'm not gonna rewrite the script now, the better path will be to use an ansible task to handle this.

whowutwut commented:
Added the last 4 commits which do the following: 
* create a playbook to replace the quay.io path for the operator
* remove ansible-lint and flake8 for now, so we can get this linting successful again 
* remove references to the change_deploy_image.py to the .yml file (and deleted it)

mew2057 commented on operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.0.1/csiscaleoperators.csi.ibm.com.crd.yaml:
```diff
@@ -14,21 +15,23 @@ spec:
     plural: csiscaleoperators
     singular: csiscaleoperator
-  scope: Namespaced
+  scope: Namespacedß
```
What's this character?

mew2057 replied:
Why is  there a  `ß` character?

mew2057 commented on .travis.yml:
```diff
@@ -190,11 +190,11 @@ jobs:
-        - pip install sphinx sphinx_rtd_theme recommonmark 
-        - cd ${TRAVIS_BUILD_DIR}/docs
-        - make html
+    # - name: "Build - Operator: docs"
```
Doc build commented out.

whowutwut commented:
HAHA! Thanks @mew2057!  Finally...
